git/git.go

```diff
@@ -49,9 +50,45 @@ func Add(files []string) {
 	}
 }
 
+func getStaged() []string {
+	out, err := exec.Command("git", "diff", "--name-only", "--staged").Output()
+	lib.Check(err)
+	staged := strings.Split(string(out), "\n")
+	var ret []string
+	for i := range staged {
+		curr := strings.TrimSpace(staged[i])
+		if curr != "" {
+			ret = append(ret, curr)
+		}
+	}
+	return ret
+}
+
+func confirmCommit(stdin io.Reader) bool {
+	fmt.Printf("Would you still like to commit? ('y' or 'n')> ")
+	reader := bufio.NewReader(stdin)
+	response, err := reader.ReadByte()
+	lib.Check(err)
+	return (response == 'y')
+}
+
 // Commit performs a git commit on added files
 // * tested
+// ! reminders untested
 func Commit(commitMessage string) {
```

Comment:
I don't see where you actually change to a specific project. The functions above don't take in a project name or path. Is there a code segment that switches the context to a specific project?

Reply:
I don't take in a pathname because I wanted to keep functionality consistent with the standard git functionality. So instead of doing 

`canaveral commit [pathname] [filename]`

The intended usage would be

```
>cd [project]
>canaveral add .
>canaveral commit -m "message"
```

As this is how the git functionality works, and I think it would be the most intuitive for users. Although if you think it's worth it I can maybe implement a wrapper or something so that if a project name is specified it performs an init / add / commit in the specified project path, and if not it performs it in the current directory. Lmk your thoughts on that

Reply:
I'm going to lean towards allowing the user to specify a project to add/commit. This is because Canaveral (so far) is designed such that you don't need to be in a specific directory to work with your projects. In other words, Canaveral functionality is the same wherever you go.

To keep consistent with this, I think something should be changed with the add and commit features. Either redesigning the features to take in a project name, or removing the features, or some other solution.

Reply:
What about adding a flag like -p [projectname] and if you specify that it performs the command for that project? I guess I don’t really see the strong benefit of being able to git add / commit from a different directory tbh, at least not as default behavior

Reply:
Yea sure, the flag idea is a good solution